binder/postBuild

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright (c) Jupyter Accessibility Team.
+# Distributed under the terms of the Modified BSD License.
+
+""" Perform a development install of jupyterlab_accessible_themes
+
+    On Binder, this will run _after_ the environment has been fully created from
+    the environment.yml in this directory.
+
+    This script should also run locally on Linux/MacOS/Windows:
+
+        python3 binder/postBuild
+"""
+import subprocess
+import sys
+from pathlib import Path
+
+ROOT = Path.cwd()
+
+
+def _(*args, **kwargs):
+    """Run a command, echoing the args
+
+    fails hard if something goes wrong
+    """
+    print("\n\t", " ".join(args), "\n")
+    return_code = subprocess.call(args, **kwargs)
+    if return_code != 0:
```

Comment:
This was part of the template I used. I think they do it to return the complete stacktrace into the console in case there's an error and not only the error.